threedworld-mit/tdw
# Resonance Audio upgrade

Closes #200 

Fixed: Resonance Audio doesn't work on OS X.

How to test:

- Download, extract, and run [this test build](https://www.dropbox.com/s/hbanj57wvrfb4hu/TDW_OSX_resonance_audio.tar.gz?dl=0); it has an upgraded version of Resonance Audio.
- Run [this version of audio.py](https://raw.githubusercontent.com/threedworld-mit/tdw/90324db9f78b8ee471a46ce52b97e79e98de8d51/Python/example_controllers/audio.py), which sets `launch_build=False`

Expected result:

- The build doesn't crash
- There are no errors or exceptions in the player log
